examples/computer_administrators_script.py

```diff
@@ -0,0 +1,57 @@
+"""
+Script for display user groups and users in group Administrators
+
+Requirements: Python 2.7 or 3.4, pyWin32, pywinauto 0.5.3+
+ download the repo: https://github.com/pywinauto/pywinauto
+ place the script to the repo root folder
+"""
+from subprocess import Popen
+from pywinauto import application
+from pywinauto.timings import always_wait_until_passes
+
+
+def is_user_an_admin():
+    import os
+    if os.name == 'nt':
+        try:
+            # only windows users with admin privileges can read the C:\windows\temp
+            os.listdir(os.sep.join([os.environ.get('SystemRoot', 'C:\\windows'), 'temp']))
+        except:
+            return os.environ['USERNAME'], False
+        else:
+            return os.environ['USERNAME'], True
+    else:
+        if 'SUDO_USER' in os.environ and os.geteuid() == 0:
+            return os.environ['SUDO_USER'], True
+        else:
+            return os.environ['USERNAME'], False

```

Comment:
I think the function returns two values: username and a boolean but you don't check it